pennylane/__init__.py

```diff
@@ -70,7 +73,6 @@
     ~_device.Device
     ~_device.DeviceError
     device
-    expval
```

Comment:
Depends entirely if we want the user to also be able to access it via `qml.measure.expval`. For backwards compatibility, we need `expval` to be top level, so to avoid having two ways of doing the same thing, I vote not to import `measure` to the top level.

Reply:
thumb (for now)